src/components/articles/ArticleCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const ArticleCard = ({ article }) => {
+  return (
+    <div className='card'>
```

Comment:
we could add this classname to the <li> and it would get rid of the need for the outer div.